src/pages/contact.css

```diff
@@ -0,0 +1,87 @@
+.contact-box {
+  background-color: #E9F3E7;
+  padding: 0;
+  margin: 10vh;
+  height: calc(120vh - 160px);
+  display: flex;
+  overflow: hidden;
+}
+
+.box-left {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 20px;
+}
+
+.title {
+  color: #3B2B23;
+  font-size: 32px;
+  font-family: 'Averia Sans Libre', sans-serif;
+  font-weight: 700;
+  line-height: 1.4;
+  text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.2);
+  margin: 25px 0 20px 0;
+  text-align: center;
+}
+
+.box-right {
+  flex: 1;
+  background-image: url('../images/contact-us-photo.png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 0;
+  padding: 0;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 500px;
+  gap: 10px; 
+}
+
+form label {
+  font-family: 'Averia Sans Libre', sans-serif;
+  color: #3B2B23;
+}
+
+form input,
+form textarea {
+  padding: 10px;
+  background-color: white;
+  border: 1px solid white;
+  border-radius: 5px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+  resize: none; /* Disables resizing of the textarea */
```

Comment:
Can you remove these comments? I don't think they are necessary for the reader as you can just look up the properties